servo/connectors/prometheus.py

```diff
@@ -819,7 +819,7 @@ async def check(
         )
 
     @servo.on_event()
-    def describe(self) -> servo.Description:
+    def describe(self, **kwargs) -> servo.Description:
```

Comment:
Why is this being updated when control and environment aren't used by this event handler?

Reply:
(see reply to above comment)